From 1a549efefb590a83b2516dfd287dbda4e8d6b846 Mon Sep 17 00:00:00 2001
From: Ovchinnikov Dmitrii <ovchinnikov.dmitrii@gmail.com>
Date: Tue, 9 Mar 2021 22:05:07 +0300
Subject: [PATCH] libavcodec/amfenc_hevc: Recommend values for min and max
 video quantizer scale.

Current settings makes bitrate larger than expected, more information:
https://github.com/HandBrake/HandBrake/issues/3447#issue-820490736

Signed-off-by: James Almer <jamrial@gmail.com>

diff --git a/libavcodec/amfenc_hevc.c b/libavcodec/amfenc_hevc.c
index bb224c5fec..a1225c09e3 100644
--- a/libavcodec/amfenc_hevc.c
+++ b/libavcodec/amfenc_hevc.c
@@ -298,6 +298,8 @@ static const AVCodecDefault defaults[] = {
     { "b",          "2M"  },
     { "g",          "250" },
     { "slices",     "1"   },
+    { "qmin",       "-1"  },
+    { "qmax",       "-1"  },
     { NULL                },
 };
 static const AVClass hevc_amf_class = {
